lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -114,10 +115,10 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
         rpcServer.resolveGroupFetch(groups);
       } catch (e) {
         handleError(e, 'error-fetch');
-      } finally {
-        decFetchCount();
       }
     }
+    // ignore errorState changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This lint suppression and the one below isn't needed since the effect doesn't read `errorState`. It does update `errorState`, but state setters don't need to be listed as dependencies since the setter function never changes.